app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -54,6 +54,10 @@
       { key: 'Study mode', value: @application_choice.course_option.study_mode.humanize },
     ]) %>
 
+    <% if FeatureFlag.active?('training_with_a_disability') %>
+      <h2 class="govuk-heading-m govuk-!-margin-top-8" id="disabilities">Disability and other needs</h2>
```

Comment:
Have added the heading for this question to indicate where it should appear in the provider interface.

If no needs are disclosed, we should show ‘Not specified’.